### ISSUE TITLE
Accessibility enhancements

### ISSUE THREAD

**Issue by gsin11** [KhalisFoundation/sttm-web#1400]
**Describe the bug**
After running Wave (accessibility tool) I found some important errors which are breaking accessibility guidelines.

**To Reproduce**
1. Install Wave accessibility toolbar
2. Go to https://www.sikhitothemax.org/help or https://www.sikhitothemax.org/hukamnama
3. Now run wave toolbar
4. You will be presented with a list of errors.

**Expected behavior**
There shouldn't be any contrast and or missing `alt` text error on images.

**Screenshots**
![image](https://user-images.githubusercontent.com/3865313/115737282-48610980-a38c-11eb-9a47-a3d72aaa6c0f.png)
![image](https://user-images.githubusercontent.com/3865313/115770260-c33a1c80-a3ac-11eb-8462-1ab980898672.png)

**Desktop (please complete the following information):**
Any browser on Desktop

**Smartphone (please complete the following information):**
Any browser on Mobile

**Additional context**
https://www.w3.org/WAI/perspective-videos/contrast/
https://www.w3.org/WAI/test-evaluate/preliminary/
